go/s3/README.md

```diff
@@ -71,11 +414,11 @@ PASS
 ok      PATH 6.593s
 ```
 
-If you want to see any log messages, enter:
+If you want to see any log messages, enter the following:
```

Comment:
**To see** any log messages, enter the **following.**

Reply:
Fixed